sdk/translation/ai-translation-text/test/public/utils/recordedClient.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Context } from "mocha";
+import { Recorder, RecorderStartOptions, env, isPlaybackMode } from "@azure-tools/test-recorder";
+import {StaticAccessTokenCredential} from "../StaticAccessTokenCredential";
+import TextTranslationFactory, { TextTranslationClient, TranslatorCredential } from "../../../src";
+import { ClientOptions } from "@azure-rest/core-client";
+import {
+  createDefaultHttpClient,
+  createPipelineRequest,
+} from "@azure/core-rest-pipeline";
+import { TokenCredential } from "@azure/core-auth";
+
+const envSetupForPlayback: Record<string, string> = {
+  TEXT_TRANSLATION_API_KEY: "fakeapikey",
+  TEXT_TRANSLATION_ENDPOINT: "https://fakeEndpoint.cognitive.microsofttranslator.com",
+  TEXT_TRANSLATION_CUSTOM_ENDPOINT: "https://fakeCustomEndpoint.cognitiveservices.azure.com",
+  TEXT_TRANSLATION_REGION: "fakeregion",
+};
+
+const recorderEnvSetup: RecorderStartOptions = {
+  envSetupForPlayback,
+};
+
+export async function startRecorder(context: Context): Promise<Recorder> {
+  const recorder = new Recorder(context.currentTest);
+  await recorder.start(recorderEnvSetup);
+  return recorder;
+}
+
+export async function createTranslationClient(options: {
+  recorder?: Recorder;
+  clientOptions?: ClientOptions;
+}): Promise<TextTranslationClient> {
+  const { recorder, clientOptions = {} } = options;
+  const updatedOptions = recorder ? recorder.configureClientOptions(clientOptions) : clientOptions;
+  const endpoint =
+    env.TEXT_TRANSLATION_ENDPOINT || "https://fakeEndpoint.cognitive.microsofttranslator.com";
```

Comment:
Could we use `assertEnvironmentVariable` from the recorder to load env vars instead of assigning default values?

Reply:
OK, done